File: TI1.4 RESTFUL_DOTNET_SINBDD_GR06/01 SERVIDOR/ConUni_RestFul_Dotnet_GR06/ConUni_RestFul_Dotnet_GR06/Modelos/Usuario.cs
namespace ConUni_RestFul_Dotnet_GR06.Modelos
{
    /// <summary>
    /// Modelo de usuario para autenticación
    /// </summary>
    public class Usuario
    {
        public string Username { get; set; }
        public string Password { get; set; }

        public Usuario()
        {
        }

        public Usuario(string username, string password)
        {
            Username = username;
            Password = password;
        }

        public override string ToString()
        {
            return $"Usuario{{username='{Username}'}}";
        }
    }
}

File: Deberes_PP_Arquitectura-entregable/TI1.3 RESTFUL_JAVA_SINBDD_GR06/02 CLICON/CLICON_RestFull_Java_GR06/src/main/java/ec/edu/monster/cliente/modelo/Usuario.java

package ec.edu.monster.cliente.modelo;


// Este POJO debe coincidir EXACTAMENTE con el del servidor
public class Usuario {
    private String username;
    private String password;
    
    // IMPORTANTE: Constructor vacío para deserialización
    public Usuario() {
    }

    public Usuario(String username, String password) {
        this.username = username;
        this.password = password;
    }

    // Getters y Setters
    public String getUsername() { return username; }
    public void setUsername(String username) { this.username = username; }
    public String getPassword() { return password; }
    public void setPassword(String password) { this.password = password; }
    
    // ¡Eliminamos el campo 'nombre' y sus getters/setters!
}